.github/workflows/regenerate-newsroom-videos.yml

```diff
@@ -0,0 +1,31 @@
+name: List everyday latest list of Newsroom Videos
+
+on: 
+  workflow_dispatch:
+  schedule:
+  #every day at midnight
+  - cron: "10 0 * * *"
+
+jobs:
+  meetings:
+      if: github.repository == 'asyncapi/website'
```

Comment:
Do we need that?

Reply:
Yes, we need it because this particular Github action should not run on any other fork repositories of the contributors.

Reply:
Yeah, you're right! Good idea with this line!